spec/helpers/location_helper_spec.rb

```diff
@@ -0,0 +1,67 @@
+# See deprecated test/helpers/location_helper_test.rb
+
+require "rails_helper"
+
+RSpec.describe LocationHelper, type: :helper do
+  describe "#sanitize_name" do
+    it "sanitizes a name" do
+      name = "California, USA; DROP TABLE USERS; <div></div>"
+      result = LocationHelper.sanitize_name(name)
+      expect(result).to eq("California, USA DROP TABLE USERS divdiv")
+    end
+  end
+
+  describe "#truncate_name" do
+    it "truncates a name" do
+      name = "Chania, Chania Municipality, Chania Regional Unit, Region of Crete, Crete, 73135, Greece"
+      result = LocationHelper.truncate_name(name)
+      expect(result).to eq("Chania, Chania Municipality, Chania Regional ...")
+    end
+
+    it "doesn't truncate shorter names" do
+      name = "Alaska, USA"
+      result = LocationHelper.truncate_name(name)
+      expect(result).to eq("Alaska, USA")
+    end
+  end
+
+  # TODO: These tests would be improved with Factories and SamplesHelper tests
+  describe "#sample_dimensions" do
+    it "gets and formats sample location dimensions" do
+      sample_ids = [1, 2, 3]
+      field_name = "collection_location_v2"
+      mock_filtered = { [nil, "Alaska, USA"] => 1, [nil, "Chania, Chania Municipality, Chania Regional Unit, Region of Crete, Crete, 73135, Greece"] => 1, [nil, "Hanoi, Vietnam"] => 1, [nil, "Redwood City, San Mateo County, California, USA"] => 112, [nil, "Zimbabwe"] => 1 }
```

Comment:
I would put each option on its own line for readability.

Reply:
👍